guide/README.md

```diff
@@ -0,0 +1,285 @@
+gmx mdrun -v # runs MD integrator. The trajectory is saved to traj.trr
+```
+
+Map this short atomistic trajectory to a CG trajectory using `csg_map`. (adjust the corresponding line in the `run.sh` script to map `traj.trr` onto `traj_cg.trr` instead of `conf(out).gro` to `conf_cg.gro`.)
+
+
+# Iterative Boltzmann inversion (IBI) for SPC/E water
+
+Here, a one-site coarse-grained (CG) model of a rigid 3-site water molecule (SPC/E model) is constructed (see the previous section of the tutorial) using the iterative Boltzmann inversion (IBI) method. The center of the CG bead is chosen to be the center of mass (COM) of a molecule. The CG radial distribution function (RDF) of the atomistic reference trajectory is then calculated using the CG bead coordinates. In the last step a coarse-grained potential is obtained by matching the RDFs of the atomisitc and CG systems using the IBI method. For a more detailed description, look at the following [publication](http://pubs.acs.org/doi/abs/10.1021/ct900369w).
+
+## Atomistic simulation
+
+A short atomistic MD simulation has been already run in the last part of the previous section (see folder:  `csg-tutorials/spce/atomistic/`). You can extract all thermodynamic information (total energy, kinetic energy, pressure, etc.) from the binary `ener.edr` file using `gmx energy`. Running the `Extract_Energies.sh` script does this for you (see the corresponding command line in the `run.sh` script). It creates an additional subfolder energies with all thermodynamic information:
+
+```bash
+./Extract_Energies.sh $equi #  The argument $equi is optional. If provided, analysis will start at the corresponding time frame (in GROMACS units, ps) (e.g. 1 to 5)
+```
+
+## Calculation of RDF
+
+Once again, check the mapping file `water.xml`. Atom names listed in the definition of the COM bead should correspond to those used in the `conf.gro` file. You can use `csg_dump` to check this:
+
+```bash
+csg_dump --top topol.tpr 
+```
+
+Check the options file `settings.xml`. It contains the section of the corresponding non-bonded interaction (`CG-CG`). After this, calculate the center of mass RDF:
+
+```bash
+csg_stat --top topol.tpr --trj traj.trr --cg water.xml --options settings.xml (--nt 3 --begin $equi) # ( ) denotes additional options: --nt # number of threads to run calculation in parallel with more then one threads (e.g. 3), --begin # time frame in GROMACS units (ps) to start analysis (e.g. 1 to 5)
+```
+Compare your RDF with the one of `CG-CG.dist.tgt` in `csg-tutorials/spce/ibi/` which has been calculated with a much longer atomistic simulation run.
+
+
+## Running IBI
+
+Now switch to the folder: `csg-tutorials/spce/ibi/`. Reduce the number of MD steps in `grompp.mdp`. Additionally, adjust the equilibration time in the settings.xml file (`cg.gromacs.equi_time`) to a lower value (time frame in GROMACS units (ps) at which the analysis of the CG trajectory is started in each IBI step). Finally, start the IBI iterations:
```

Comment:
"Reduce the number of MD steps in `grompp.mdp`. Additionally, adjust the equilibration time ..." - > "Reduce the number of MD steps in `grompp.mdp` and adjust the equilibration time ..."